### ISSUE TITLE
Size from config file not used 

### ISSUE THREAD

**Issue by AndydeCleyre** [bil-elmoussaoui/Hardcode-Tray#388]
### Specifications
- Hardcode-Tray version : `3.9.r8.gbbab5c2`
- Distribution : `Arch`
- Gtk version : `3.22.16`
- Inkscape version : `0.92.1`
- Cairo version : `1.14.10`
- ~~Sni-qt~~
- Application name :  `Telegram Desktop`
- Application version : `1.1.7`

### Expected behavior and actual behavior
The size specified in the config file should be used when none is specified in the command arguments. Size 22 is used.

### Steps to reproduce the problem
1. Write config file `~/.config/hardcode-tray.json`:

```json
{
    "icons": {
        "theme": "Papirus-Dark",
        "size": 64
    },
    "nwjs": "/opt/nwjs-sdk/"
}
```

2. Modify source to allow the sizes used by your desktop environment to be used by hardcode-tray:

```bash
sudo sed -Ei 's/^ICONS_SIZE.+/ICONS_SIZE = [16, 22, 24, 32, 48, 64, 128]/' /opt/Hardcode-Tray/src/const.py
```
 
3. Run hardcode-tray:

```bash
hardcode-tray -a -cc "#d3dae3 #c2b790" "#5294e2 #a94d37"
# . . .
# Icon Size: 22
# . . .
```

4. Run hardcode-tray but override the size:

```bash
hardcode-tray -a -cc "#d3dae3 #c2b790" "#5294e2 #a94d37" --size 64
# . . .
# Icon Size: 64
# . . .
```

**Comment by bil-elmoussaoui** [comment]
I will take a look

**Comment by AndydeCleyre** [comment]
And do I understand correctly that it's a deliberate decision to require that users modify the source of your program (step 2 above) after every update in order to support Plasma icon sizes, as a safety measure? I would love if hardcode-tray supported Plasma out of the box . . .

**Comment by bil-elmoussaoui** [comment]
Fixed on master. For the icons sizes, I will do some tests and if everything scales perfectly, i will allow users to use icons sizes bigger than 24px.

:stop_sign: 
Don't update to master version yet, as i'm trying to fix this #385. A new version should land this evening
